lib/manageiq/graphql/field_name_camelizer.rb

```diff
@@ -0,0 +1,20 @@
+module ManageIQ
+  module GraphQL
+    class FieldNameCamelizer
+      def instrument(_type, field)
```

Comment:
Would it be easy to add isolated tests for this?

Reply:
I'm not sure actually but I'd love the challenge; I assume we can request the introspection metadata somehow.

Reply:
Seriously though if it's a pain at all I'm fine with the integrated test coverage :+1: